REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
this should be ZMQ 0.3.4.
it's been changed with `unsafe_string(::Message)` since ZMQ 0.3.3,
but ZMQ 0.3.3 has wrong(conditional deprecation).